app/models/word.rb

```diff
@@ -1,11 +1,16 @@
 class Word < ApplicationRecord
 	belongs_to :category
 	
-	# has_many :answers
-	# has_many :choices
+	has_many :answers
+	has_many :choices, dependent: :delete_all
 
-	# accepts_nested_attributes_for :choices
-	# accepts_nested_attributes_for :answers
+	accepts_nested_attributes_for :choices
+	accepts_nested_attributes_for :answers
 
 	validates :content, presence: true
+	validates :correct, acceptance: true
```

Comment:
You can add custom validation here in the `Word` model, because you can fetch all choices from here.
Reference: https://guides.rubyonrails.org/active_record_validations.html#custom-methods